src/components/Footer/index.js

```diff
@@ -1,35 +1,123 @@
 import React from 'react'
 import SocialIcons from '../SocialIcons'
+import { t, Trans } from '@lingui/macro'
 import './styles.scss'
 
 const Footer = () => (
   <footer className='footer' role='contentinfo'>
     <div className='footer-primary'>
       <div className='wrapper'>
         <div className='container'>
-          <h2 className='footer-primary__title'>Rockefeller <span className='line-break'>Archive Center</span></h2>
+          <h2 className='footer-primary__title'>
+            <Trans comment='Primary Title'>
+              Rockefeller
+              <span className='line-break'>
+                Archive Center
+              </span>
+            </Trans>
```

Comment:
Do you want each individual connected item to be translated, or do you want the whole file wrapped by a Trans to convert whole components at a time?

Reply:
I think for the most part individual unconnected items is better, but some sections like the Social Icons could probably be one huge Wrappings since there are so few actual strings that need translated and they all seem related.

Reply:
I think that's right - if someone were going to add or remove one of these social icons they'd have to make changes to the HTML structure anyway.